### PR TITLE
[SENDMAIL] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/shellext/sendmail/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,14 +1,14 @@
 /*
- * PROJECT:     sendmail
+ * PROJECT:     ReactOS Sendmail
```
Comment (binarymaster) [inline dll/shellext/sendmail/lang/ro-RO.rc:2]: Project name was correct
```suggestion
 * PROJECT:     sendmail
```

File: dll/shellext/sendmail/lang/ro-RO.rc [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
  * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
  * PURPOSE:     Romanian resource file
- * TRANSLATORS: Copyright 2022 Andrei Miloiu <miloiuandrei@gmail.com>
+ * TRANSLATORS: Copyright 2022-2024 Andrei Miloiu <miloiuandrei@gmail.com>
```
Comment (binarymaster) [inline dll/shellext/sendmail/lang/ro-RO.rc:5]:
```suggestion
 * TRANSLATOR:  Copyright 2022-2024 Andrei Miloiu <miloiuandrei@gmail.com>
```

**Comment by AndreiMiloiu** [comment]
Thank you @binarymaster!

**Pull request merged**